README.md

```diff
@@ -78,9 +80,9 @@ yarn install
 ```
 3. Copy iohook binaries
 ```sh
-mkdir -p node_modules\iohook\builds\electron-v80-win32-x64\build\Release\
-cp iohook\iohook.node node_modules\iohook\builds\electron-v80-win32-x64\build\Release\
-cp iohook\uiohook.dll node_modules\iohook\builds\electron-v80-win32-x64\build\Release\
+mkdir -p node_modules/iohook/builds/electron-v80-win32-x64/build/Release/
```

Comment:
Since you added a `copy-io.sh` script, you might as well replace these lines with `sh ./copy-io.sh`

Reply:
This is true. Should have done that. Thanks!